SPECS/libXaw/libXaw.spec

```diff
@@ -54,23 +54,26 @@ export CFLAGS="%{optflags} -Os"
 make %{?_smp_mflags}
 
 %install
-
 make install DESTDIR=%{buildroot} INSTALL="install -p"
+install -pm 644 COPYING README.md ChangeLog %{buildroot}/%{_pkgdocdir}
 find %{buildroot} -type f -name "*.la" -delete -print
 
 %post -p /sbin/ldconfig
 %postun -p /sbin/ldconfig
 
 %files
 %license COPYING
-%doc ChangeLog README
+%doc ChangeLog README.md
+%dir %{_pkgdocdir}
```

Comment:
Why was this moved from the devel subpackage to the main package?

Reply:
the publisher made changes to their reference SPEC, and I followed suit.